amundsen_application/static/js/components/HomePage/tests/index.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { mapDispatchToProps, HomePage, HomePageProps } from '../';
+
+import SearchBar from 'components/SearchPage/SearchBar';
+import BookmarkList from 'components/common/Bookmark/BookmarkList';
+import PopularTables from 'components/common/PopularTables';
+
+describe('SearchBar', () => {
+  const setup = (propOverrides?: Partial<HomePageProps>) => {
+    const props: HomePageProps = {
+      searchReset: jest.fn(),
+      history: {
```

Comment:
worthy mention: I'll see if i have time to look into how to better write tests that leverage the router. This works but I recognize it's pretty inconvenient.